Calypso-SystemQueries.package/ClyProjectManagerRegistry.class/instance/projectQueryFor..st

```diff
@@ -0,0 +1,7 @@
+queries
+projectQueryFor: aScope
```

Comment:
I don't know. Do you mean hardcoding in the AllProjects query how to fetch the projects? I like the idea that we can compose them dynamically also.

In any case, again, this improves what was done before by a lot, because now we can have have for real different kind of managers, retrieveing different kind of objects, and this works. We can enhance it later.

Reply:
It was lost in the changes we did, I'd like to understand how to put it back.